com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Utility/Logic/AndNode.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class AndNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,     // Version
+            "And", // Name
+            "Out = A && B;",
+            new ParameterDescriptor("A", TYPE.Bool, Usage.In),
+            new ParameterDescriptor("B", TYPE.Bool, Usage.In),
+            new ParameterDescriptor("Out", TYPE.Bool, Usage.Out)
+        );
+
+        public static Dictionary<string, string> UIStrings => new()
+        {
+            { "Category", "Utility, Logic" },
+            { "Name.Synonyms", "&" },
```

Comment:
Synonym should probably be `&&`. This single `&` is for bitwise And, which might be something we'll want to expose later.

Reply:
Done